src/apiClient.ts

```diff
@@ -42,9 +67,33 @@ class ApiClient {
     })
   }
 
-  public setTokens(tokens: Tokens): void {
-    this.tokens.accessToken = tokens.accessToken
-    this.tokens.refreshToken = tokens.refreshToken
+  public getConfiguration() {
+    return {
+      ...this.configuration,
+      configured: Object.keys(this.configuration).length > 0,
+      version: this.version,
```

Comment:
can this default be merged when the api client is initialized? looking at `this.configuration` at runtime might yield a different result as when calling `getConfiguration`. we're opening ourselves to potential mishaps with this

Reply:
Not sure where the function is consumed but kept it as is, just moved it form the index to the class as I removed the wrapper....
Krzysztof will tackle the configuration in his PR so it might be a good opportunity to update there...

Reply:
cc @Averethel